javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,14 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  var newArray = [];
+  var i = values.length;
+  while (i > 0) {
+    if (typeof (values[i - 1]) === typeof (7)) {
+      newArray.unshift(values[i - 1]);
+      i--;
+    } else {
+      i--;
+    }
+  }
```

Comment:
Is this just for practice with `while` loops? If so, I would stick to keeping this a positive movement through the array, going backwards through the array adds an unnecessary level of complexity to the code.